packages/next/build/webpack.js

```diff
@@ -172,7 +176,18 @@ export default async function getBaseWebpackConfig (dir, {dev = false, isServer
   const defaultLoaders = {
     babel: {
       loader: 'next-babel-loader',
-      options: {dev, isServer, cwd: dir}
+      options: {
+        dev,
+        isServer,
+        cwd: dir,
+        cacheIdentifier: JSON.stringify({
+          config,
+          'next': NEXT_VERSION,
+          '@babel/core': babelPkg.version,
+          '@babel/loader': babelLoaderPkg.version
```

Comment:
Guess this line has to be removed since we have babel-loader inside next after this PR.